### ISSUE TITLE
c-75 focus outline appears on second attempt

### ISSUE THREAD

**Issue by kineotesting** [adaptlearning/adapt_framework#1062]
After failing the first attempt, as soon as you click 'Reset' for the second attempt an orange focus outline appears around the slider button.

![1](https://cloud.githubusercontent.com/assets/19309048/15246300/a199f552-1903-11e6-9cd6-22704d0d3d28.jpg)

**Comment by lc-thomasberger** [comment]
input[type="range"]:focus

check why focus is set
show outline only when accessibility is turned on?

**Comment by kineotesting** [comment]
Fixed
